frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -377,7 +414,11 @@ export function Experiment(): JSX.Element {
                         <Col span={14}>
                             <div>
                                 Test that your code works properly for each variant:{' '}
-                                <a href="https://posthog.com/docs/user-guides/feature-flags#develop-locally">
+                                <a
+                                    target="_blank"
+                                    rel="noopener noreferrer"
```

Comment:
why `noopener` and `noreferrer` ?

Reply:
oh so it would open into a new tab instead of redirecting the user on the current one, away from their experiment

Reply:
ah, you just need target blank for that.

Reply:
I just put it in because of this https://web.dev/external-anchors-use-rel-noopener/ though I suppose it should be fine without it too since we're linking to our own website 😊

Reply:
Ah, right. We do trust our website and want to increase its referral score, so I'd say it's good to have it in